src/core/Kokkos_RemoteSpaces_ViewMapping.hpp

```diff
@@ -1204,11 +1204,22 @@ class ViewMapping<Traits, Kokkos::Experimental::RemoteSpaceSpecializeTag> {
     }
 #endif
 
+    const std::string &alloc_name =
+        static_cast<Kokkos::Impl::ViewCtorProp<void, std::string> const &>(
+            arg_prop)
+            .value;
```

Comment:
Why did you declare this outside of the `if`?

Reply:
Fixed.